### PR TITLE
Feature/escrowless

### PR DESCRIPTION
**Pull request by 0xabstracted**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,14 @@ target
...
+xNFT_Default
+xNFT_Tim
+ticket-xnft
+client
+Cargo.lock
+package-lock.json
+package.json
+yarn.lock
+
+
```
Comment (cotemper) [inline .gitignore]: Why are they included in `.gitignore` file?

File: .solitarc.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const path = require("path");
-const programDir = path.join(__dirname, "..", "splitwave");
-const idlDir = path.join(__dirname, "idl");
-const sdkDir = path.join(__dirname, "src", "generated");
-const binaryInstallDir = path.join(__dirname, ".crates");
+const programDir = path.join(__dirname, "./programs/splitwave");
+const idlDir = path.join(__dirname, "./client/idl");
+const sdkDir = path.join(__dirname, "./client/src", "generated");
+const binaryInstallDir = path.join(__dirname, "./client/.crates");
```
Comment (cotemper) [inline .solitarc.js]: We use `app` directory for solita sdk and I don't think we need to change this solita configuration.

File: package.json [hunk 3]
```diff
@@ -11,8 +11,10 @@
     "amman:stop": "./node_modules/.bin/amman stop"
   },
   "dependencies": {
+    "@coral-xyz/xnft-cli": "^0.6.8-latest.5333",
     "@project-serum/anchor": "^0.26.0",
-    "anchor-lang": "^0.0.1-security"
+    "anchor-lang": "^0.0.1-security",
+    "react-xnft": "^0.2.0-latest.3375"
```
Comment (cotemper) [inline package.json]: I don't think xNFT packages are used in this repo.

File: Anchor.toml [hunk 4]
```diff
@@ -10,7 +10,7 @@ url = "https://api.apr.dev"
 [provider]
 cluster = "https://rpc-devnet.helius.xyz/?api-key=ec0c9707-e412-45e4-9608-5a4cb6141af9"
 #cluster = "http://localhost:8899"
-wallet = "/home/abstracted/ant_labs/wallets/TempUpdateAuthority/TUAXRFzyLeXmG9wPLaMXt66jUagfrWmL9oGq4rMwjAu.json"
+wallet = "/Users/0xabstracted/Documents/wallets/TempUpdateAuthority/TUAXRFzyLeXmG9wPLaMXt66jUagfrWmL9oGq4rMwjAu.json"
```
Comment (cotemper) [inline Anchor.toml:13]: What is this change?

File: programs/splitwave/src/instructions/create_splitwave_id.rs [hunk 5]
```diff
@@ -11,10 +11,11 @@ pub struct CreateSplitwaveId<'info> {
         seeds = [SEED_SPLITWAVE_ID],
         bump
     )]
-    splitwave_id: Account<'info, SplitwaveId>,
+    splitwave_id: Box<Account<'info, SplitwaveId>>,
 
     #[account(mut)]
     payer: Signer<'info>,
+    pub rent: Sysvar<'info, Rent>,
```
Comment (cotemper) [inline programs/splitwave/src/instructions/create_splitwave_id.rs:18]: Why is this "pub" only added to this place? Can we use the same coding style in codebase?

**Review by cotemper — CHANGES_REQUESTED** [review]
Left some comments. And I don't see any "escrowless" related codes even though the branch name is "Feature/escrowless".

File: Anchor.toml (hunk 4 above)
Comment (0xabstracted) [reply Anchor.toml:13]: it is because I shifted from linux to mac

File: programs/splitwave/src/instructions/create_splitwave_id.rs (hunk 5 above)
Comment (0xabstracted) [reply programs/splitwave/src/instructions/create_splitwave_id.rs:18]: fixed in next commit

File: package.json (hunk 3 above)
Comment (0xabstracted) [reply package.json]: fixed in next commit

File: .gitignore (hunk 1 above)
Comment (0xabstracted) [reply .gitignore]: fixed in next commit

File: .gitignore [hunk 6]
```diff
@@ -6,3 +6,10 @@ target
 **/*.rs.bk
 node_modules
 test-ledger
+.crates
+client
+Cargo.lock
+package-lock.json
+package.json
+yarn.lock
+yarn-error.log
```
Comment (cotemper) [inline .gitignore]: Can we revert these changes?
